cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/cluster/AbstractClusterOperations.java

```diff
@@ -265,26 +269,38 @@ public final void reconcile(String namespace, String name) {
                                     log.error("Failed to add {} cluster {}.", clusterDescription, nameFromCm);
                                 }
                                 lock.release();
+                                log.debug("Lock released");
                             });
                         }
                     } else if (resources.size() > 0) {
+
+                        List<Future> result = new ArrayList<>(resources.size());
```

Comment:
`List<Future<Void>>`

Reply:
`CompositFuture` works with `List<Future>` otherwise it complains.

Reply:
Oh, yeah that's true. I remember now.

Reply:
@tombentley so can I merge it or you have other comments ?